client/src/boot/index.js

```diff
@@ -10,7 +10,7 @@ window.document.addEventListener('DOMContentLoaded', () => {
 
   const element = window.document.getElementById('mfa-app');
   const schemaURL = element.dataset.schemaurl;
-  ReactDOM.render(<MultiFactorApp schemaURL={schemaURL} />, element);
+  ReactDOM.render(<Login schemaURL={schemaURL} />, element);
```

Comment:
Not sure I agree with this change. I'd expect a user to be able to go to the MFA endpoint at any point on the frontend, if they wanted to modify their MFA settings. It also doesn't look related to renaming the Login component

Reply:
Ok I can revert this part. Right now `MultiFactorApp` is incredibly non-descriptive. I think a name like `Login` (given it's in an MFA namespace) is more descriptive as this handles the whole registration/verification flow presented while logging in.

Reply:
I see, you've renamed that component as well. Ok I guess